torch_geometric/contrib/flag/flag.py

```diff
@@ -1,17 +1,51 @@
 import numbers
+from typing import List
 
 import torch
 
 from .flag_callback import FLAGCallback
 
+
 class FLAG(torch.nn.Module):
+    """
+    The Free Large-scale Adversarial Augmentation on Graphs (FLAG) algorithm
+    from the `Robust Optimization as Data Augmentation for Large-scale Graphs
+    <https://arxiv.org/pdf/2010.09891.pdf>`_ paper.
+
+    FLAG is a model and task free data augmentation strategy for graphs. It
+    runs `n_ascent_step` forward and backward passes, perturbing the node
+    features, performing gradient ascent on this perturbation, and
+    accumulating the gradients with respect to the model parameters. At the
+    end of the steps, the model parameters are updated with these accumulated
+    gradients.
+
+    FLAG can be used in node classification, link prediction, and graph
+    classification. However, this implementation is only meant for data with
+    continuous node features (while the original paper handles discrete node
+    features, it does so through changes to the model's `forward` function).
+
+    .. note::
+        For examples of using FLAG in a training loop, see
```

Comment:
@reislerjul Just want to make sure we actually have an example available here by the time this gets merged if we're going to leave this note in the docs.

Reply:
Good point! Let me remove this for now.